### DISCUSSION TITLE
extra meeting to go over project responsibilities

### DISCUSSION THREAD

**Discussion by Emend026** [anhourandaquarter/carmina#12] [General]
Hey guys. I suggest we meet up via zoom this week to go over the final project in greater details. We need to make sure we discuss what we need for the project and what tasks and responsibilities for us will be moving forward. Can you all send me your availability for this week (include dates & times)? I'll send over a zoom link once I have everyone's responses. Thanks!

my availability: 
Tuesday: 9AM-11AM; 1PM-5PM 
W: 1PM-5PM 
Thur: 9AM-11AM; 1PM-5PM
F: 11AM-12PM

**Comment by hhchan94** [comment]
Tuesday: 4:00 PM - 5:00 PM
Wednesday: 2:30 PM - 5:00 PM
Thursday: 9:00 AM - 11:00 AM
Friday: 11:00 AM - 12:00 PM"

**Comment by infinite0x20** [comment]
T: 4-5pm
W: none : (
Th: 9-10am
F: none : (

Thanks for starting this convo, Elizabeth and Hui-Hsuan!

**Comment by Emend026** [comment]
Zoom link for our meeting later today! 

**@ 5:30PM** 

Join Zoom Meeting
https://washington.zoom.us/j/9539034249

**Comment by infinite0x20** [comment]
@thatswhatsimonsaid meeting now!

**Comment by Emend026** [comment]
Hi guys. thanks for meeting!  @anhourandaquarter is going update our repo with the notes we discussed and also what the next steps will be. Hopefully on Thursday during or right after class we can briefly go over these updates together so we're all on the same page and we can also schedule out another zoom meeting too.

**Reply by hhchan94** [reply]
Oh just saw this! Got it! Will wait for Thursday!

**Reply by thatswhatsimonsaid** [reply]
yay

**Comment by hhchan94** [comment]
Guys, sorry to hop off early without notification. My home suddenly experienced a power outage (I guess it’s due to the strong wind), which disconnected my Wi-Fi, and even my phone had no signal for a while. When my phone finally regained signal, I tried to rejoin the session using my phone hotspot, but it seems the meeting was already over. Please let me know what the next steps are. Thank you!

**Comment by thatswhatsimonsaid** [comment]
I woke up but feeling  not so well just now, and wanted to lyk - I'm going to try to sleep my sickness off more, but wanted to give a heads up in case I don't  show up today

**Comment by infinite0x20** [comment]
Maybe we can have another meeting next Tuesday (11/26) -- what times are good? I'm available anytime 5pm and after

**Comment by infinite0x20** [comment]
We could also do tomorrow (Friday Nov. 22) or sometime on the weekend

**Reply by Emend026** [reply]
@thatswhatsimonsaid @anhourandaquarter @hhchan94  I can do either too. What's everyone's availability? 

F: 9AM-10AM; 11AM-12PM; 1:30PM-2PM; 3-5PM 
Sat & Sun: mornings preferably. 9AM-12PM

**Reply by hhchan94** [reply]
I am available on Friday 3-5. Weekend morning also works for me !

**Comment by infinite0x20** [comment]
Agenda for the meeting:
- Go over open issues + review who's working on what
- Testing requirements for `parser`
- VS Code walkthrough (if needed)
- Set ETA for priority issues (hoping to have both priority issues done before Thanksgiving)

**Comment by thatswhatsimonsaid** [comment]
3-5 on Friday works for me too!

**Reply by infinite0x20** [reply]
Augh, I'm unfortunately not available then. But I could do 10am on Saturday, if that works for others.

**Reply by thatswhatsimonsaid** [reply]
Dang it!! I'll be busy this weekend, but I can do any other time tomorrow

**Reply by Emend026** [reply]
Are you guys available on Monday then?

**Reply by infinite0x20** [reply]
I'm free on Monday 6:30pm and afterwards. *Might* be able to do 10am-11am

**Reply by hhchan94** [reply]
After 6:30 also works for me.

**Reply by Emend026** [reply]
same. 6:30 works. @thatswhatsimonsaid does 6:30 today work for you as well?

**Comment by Emend026** [comment]
zoom link for tonight 6:30pm: 

Join Zoom Meeting
https://washington.zoom.us/j/9539034249

**Reply by infinite0x20** [reply]
Hey y'all-- currently getting held up by a work thing. I might join closer to 6:45. Feel free to start ahead of me!

**Reply by thatswhatsimonsaid** [reply]
No worries - I can join at 6:45 as well then
